projects/plugins/social/src/js/components/social-module-toggle/index.tsx

```diff
@@ -59,6 +60,16 @@ const SocialModuleToggle: React.FC = () => {
 					{ __( 'Manage social media connections', 'jetpack-social' ) }
 				</Button>
 			) }
+			{ services.map( service =>
+				service.type === 'publicize' ? (
+					<ConnectButton
+						connectUrl={ service.connect_URL }
+						onClose={ res => console.log( res ) }
```

Comment:
This is where we need to check the account data we get back from the authentication process against the already connected accounts, allow the user to select the account to connect and call the API to make the connection.

We should also return errors here if necessary and feedback to the user. Currently the process can just end with no feedback, so while we're moving the connections management, it would be good to add better feedback when things go wrong or are cancelled.

Reply:
Yeah, I have changed that to `onConfirm`.